pkg/image/prlimit_test.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2018 The CDI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"bytes"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+const killedByTestError = "Had to kill process"
+
+func TestLimits(t *testing.T) {
```

Comment:
Wondering if these tests are really necessary and if so should they be functional tests?

Reply:
I am not sure we've ever found a real image in the wild that can cause problems.  Maybe we can run using a fake qemu binary that spinloops or something?